head.html

```diff
@@ -3,4 +3,4 @@
 <link id="favicon" rel="icon" type="image/svg+xml" href="/icons/adobe.svg">
 <link rel="stylesheet" href="/hlx_fonts/pnv6nym.css"/>
 <meta name="viewport" content="width=device-width, initial-scale=1"/>
-<script type="module" src="/scripts.js"></script>
+<script type="module" src="/dist/app.min.js"></script>
```

Comment:
Not sure if this still needs to be a `module`, now that the code in `dist` will be a simple IIFE.

Reply:
I'd still recommend using `defer` or `module` to make sure it's not intefering with the rest of the resoutrces being loaded.

Reply:
Sounds good, we'll make some tests and add it back.